bezier/src/cubic_bezier.rs

```diff
@@ -272,6 +332,16 @@ impl CubicBezierSegment {
         return rect(min_x, min_y, max_x - min_x, max_y - min_y);
     }
 
+    /// Returns the smallest rectangle the curve is contained in
+    pub fn minimum_bounding_rect(&self) -> Rect {
+        let min_x = self.sample(self.find_x_minimum()).x;
+        let max_x = self.sample(self.find_x_maximum()).x;
+        let min_y = self.sample(self.find_y_minimum()).y;
+        let max_y = self.sample(self.find_y_maximum()).y;
```

Comment:
A little detail, but you can write `self.sample(self.find_x_minimum()).x` into `self.sample_x(self.find_x_minimum());` to avoid computing y (not extremely important since the code is already correct).